pkg/api/types.go

```diff
@@ -360,7 +360,7 @@ type PersistentVolumeSource struct {
 	Cinder *CinderVolumeSource
 	// CephFS represents a Ceph FS mount on the host that shares a pod's lifetime
 	// +optional
-	CephFS *CephFSVolumeSource
+	CephFS *CephFSPersistentVolumeSource
```

Comment:
This does not break the API. Go type names are not persisted in JSON or protobuf. To be API compatible, all matching fields (types, json names, and protobuf tags) must be identical between the two types.